config/_default/config.toml

```diff
@@ -1,7 +1,7 @@
 baseURL = ""
 languageCode = "en-us"
 title = "GoCD User Documentation"
-uglyUrls = true
+uglyUrls = false
```

Comment:
@kritika-singh3 -- Here we are modifying the global config. This will cause the urls to change at all over the place.
Not sure if it's the safe enough change to do.

Just a question. is there a way for Hugo to spit out all the generated urls? If so, we can compare the old URLs and the new URLs to see what all has changed.

Reply:
I think it is ok to change the global config as it would mimic the original docs for all pages.

Reply:
Is there a way for us to verify?